README.md

```diff
@@ -8,26 +8,50 @@
 [![Total Downloads](https://img.shields.io/packagist/dt/graze/:package-name.svg?style=flat-square)](https://packagist.org/packages/graze/:package-name)
 
 >This is heavily inspired by the skeleton project from [The PHP League](https://github.com/thephpleague/skeleton). With some changes based on our preferences. Thanks to them!
-
+>
 >**Note:** Replace `:author-name` `:author-username` `:author-email` `:package-name` `:package-description`, `:scrutinizer-access-token`, `:year` with their correct values in [README.md](README.md), [CHANGELOG.md](CHANGELOG.md), [CONTRIBUTING.md](CONTRIBUTING.md), [LICENSE.md](LICENSE.md), [composer.json](composer.json), [Dockerfile](Dockerfile) and [travis.yml](.travis.yml) files, then delete this blockquote.
-
+>
 >**To use this skeleton project:**
-```shell
-$ git clone --depth=1 git@github.com:graze/skeleton-project.git :package-name
-$ cd ./:package-name
-$ rm -rf .git
-$ git init
-```
-
+> ```shell
+> $ git clone --depth=1 git@github.com:graze/skeleton-project.git :package-name
+> $ cd ./:package-name
+> $ rm -rf .git
+> $ git init
+> ```
+>
 >You now have a copy of the files in this repository, in a new git repository with no previous history that can you manipulate and push to other remote repositories.
-
->**Continuous Integration**
-
+>
+> ## Continuous Integration
+>
 >Your project should make use of the following remote CI services:
-- [Travis CI](https://travis-ci.org/graze/) - automated testing
-- [Scrutinizer CI](https://scrutinizer-ci.com/organizations/graze/repositories) - code quality
-
->Follow the instructions on the respective sites to add your project. Builds should be triggered on each Pull Request and each merge in to the master branch.
+>
+> ### [Travis CI](https://travis-ci.org/graze/) - automated testing
+>
+> 1. Log-in with github
+> 1. visit: https://travis-ci.org/profile/graze
+> 1. Click `sync with github`
+> 1. Enable your project
+>
+> ### [Scrutinizer CI](https://scrutinizer-ci.com/organizations/graze/repositories) - code quality
+>
+> 1. Log-in via github
+> 1. Click `+ Add Repository`
+> 1. Select `graze` as the organisation
```

Comment:
This is where I fell over. I wasn't *in* the graze organisation for this to be listed. That's where all my problems began, but I only knew that after the fact.